.eslintrc

```diff
@@ -0,0 +1,65 @@
+
+{
+  // Extend the AirBnb lint config
+  "extends": "airbnb",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "ecmaFeatures": {
+      "globalReturn": true,
+      "impliedStrict": true,
+      "jsx": true
+    },
+    "sourceType": "module",
+  },
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true,
+    "jquery": true,
+    // Optional Enables
+    "webextensions": false, // Enable if using Web Extensions
+    // Optional Testing Frameworks
+    "jasmine": false, // Enable if using Jasmine testing framework
+    "protractor": false, // Enable if using Protractor testing framework
+    "mocha": false // Enable if using Mocha testing framework
+  },
+  "globals": {
+    "jQuery": true,
+    "angular": false, // Enable if using Angular
+  },
+  // Do NOT change these rules
+  "rules": {
+    "indent": [2, 2, {"SwitchCase": 1}],
+    "max-len": [2, 80, 4, {
+      "ignoreComments": true,
+      "ignoreUrls": true,
```

Comment:
I'm going to update the base eslint, it looks like we can add an option here for `"ignoreTemplateLiterals": true`